Src/Sentry.Xamarin/SentryXamarin.cs

```diff
@@ -21,6 +21,12 @@ public static class SentryXamarin
         /// <param name="configureOptions">The configure options.</param>
         public static void Init(Action<SentryXamarinOptions> configureOptions)
         {
+            if (SentrySdk.IsEnabled)
+            {
+                Console.Error.WriteLine("SentryXamarin.Init was called again. It should only be called once. Any change to options will be ignored.");
```

Comment:
Perhaps only show the error if user enabled the debug flag?
```suggestion
                if (configureOptions.Debug) {
                    Console.Error.WriteLine("SentryXamarin.Init was called again. It should only be called once. Any change to options will be ignored.");
                }
```

Reply:
Agreed, but I might as well just use the logger from the options then.  There's a small overhead in configuring options that are going to be discarded, but I think it's not significant.